docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -21,7 +21,7 @@ image::images/view-alert-details.png[View details button, 200]
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
 
-* Visualize and group alerts by specific parameters in the visualization section. Use the menu on the left to select a view type (*Trend*, *Table*, or *Treemap*), and use the menus on the right to select the ECS fields used for grouping alerts. Refer to <<visualize-alerts>> for more on each view type.
+* Visualize and group alerts by specific parameters in the visualization section. Use the buttons on the left to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the menus on the right to select the ECS fields used for grouping alerts. Refer to <<visualize-alerts>> for more on each view type.
```

Comment:
```suggestion
* Visualize and group alerts by specific fields in the visualization section. Use the buttons on the left to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the menus on the right to select the ECS fields used for grouping alerts. Refer to <<visualize-alerts>> for more on each view type.
```
I don't think this is an accurate use of '[parameter](https://www.techopedia.com/definition/3725/parameter-param)',

Reply:
It was intended in a more non-technical sense, but since these are technical docs, I agree accuracy is better here.